app/views/admin/versions/_object_desc_and_link.html.haml

```diff
@@ -1,175 +1,225 @@
+- unless version.item_type == 'Role' || version.item_type == 'UsersRole'
+  - conference = Conference.find_by(id: version.conference_id)
+  - conference_short_title = conference.try(:short_title) || current_or_last_object_state('Conference', version.conference_id).try(:short_title) || ' '
+
 - case version.item_type
+- when 'Organization'
+  organization
+  = link_to_organization(version.item_id)
+
 - when 'UsersRole'
   - users_role = current_or_last_object_state(version.item_type, version.item_id)
-  - user = current_or_last_object_state('User', users_role.user_id)
-  = 'role'
-  = link_to users_role.role.name, admin_conference_role_path(conference_id: Conference.find(version.conference_id).short_title, id: users_role.role.name)
-  = version.event == 'create' ? 'to' : 'from'
-  = 'user'
+  - role = Role.find_by(id: users_role.role_id) if users_role
+  role
+  - if role.name == 'organization_admin'
+    / organization_admin belongs to organization and not conferencess
+    - organization = Organization.find(version.conference_id) 
+    = link_if_alive version, role.name,
+            admin_organizations_path, organization
+    / edit the above path to organization admin path after #1571 PR is merged
```

Comment:
We should use `-# a comment` instead of `/ a comment`, because `/`-comments get output to the HTML.
I see 4 places in this PR where it has to be changed.